h2o-core/src/test/java/hex/ModelBuilderTest.java

```diff
@@ -121,9 +121,25 @@ public void testMakeUnknownModel() {
       ModelBuilder.make("invalid", null, null);
       fail();
     } catch (IllegalStateException e) {
-      assertEquals("Algorithm 'invalid' is not registered. Available algos: []", e.getMessage()); // core doesn't have any algos
+      // core doesn't have any algos but depending on the order of tests' execution DummyModelBuilder might already be registered
+      assertTrue( e.getMessage().startsWith("Algorithm 'invalid' is not registered. Available algos: [")); 
     }
   }
+  
+  @Test
+  public void testMakeByModelParameters() {
+    
+    String registrationAlgoName = "dummymodelbuilder"; // by default it is a lower case of the ModelBuilder's simple name
+    DummyModelParameters params = new DummyModelParameters("dummy_MSG", Key.make( "dummyGBM_key"), registrationAlgoName);
+    new DummyModelBuilder(params, true); // as a side effect DummyModelBuilder will be registered in a static field ModelBuilder.ALGOBASES
```

Comment:
I would register this in `@BeforeClass` just to make sure this particular test method doesn't have side effects.

Otherwise, it looks good to me.

Reply:
Fine with me, this is a minor thing.